specification/sections/05-events.md

```diff
@@ -28,9 +28,17 @@ graph
 
 > The `provider` **MAY** define a mechanism for signaling the occurrence of one of a set of events, including `PROVIDER_READY`, `PROVIDER_ERROR`, `PROVIDER_CONFIGURATION_CHANGED` and `PROVIDER_STALE`, with a `provider event details` payload. 
 
+Providers cannot emit `PROVIDER_CONTEXT_CHANGED` or `RECONCILING` event.
```

Comment:
```suggestion
Providers cannot emit `PROVIDER_CONTEXT_CHANGED` or `PROVIDER_RECONCILING` event.
```

At least that's what the event is called in the types doc.

Reply:
Thanks!